README.md

```diff
@@ -1,10 +1,20 @@
 # OpenFGA RFCs
 
-Want to suggest a change to the OpenFGA project? That's great!
+Want to suggest a change to the OpenFGA project? That's great! We follow an RFC (Request for Comments) process for substantial changes to the project. An RFC is a way to propose, communicate and coordinate on new efforts for the project. 
 
-We follow an RFC (Request for Comments) process for substantial changes to the project.
+## 👀 Open Requests For Comments
 
-## When the RFC process is necessary
+Check out the PRs below to see and contribute to features the core team and community are currently evaluating. Let us know your feedback with emojis and comments.
+
+| Feature                                                                       |  PR | Status     |
+|:------------------------------------------------------------------------------|----:|:-----------|
+|[Add RFC for ExpandedWatch API](https://github.com/openfga/rfcs/pull/4)        |  #4 | Draft      |
+|[Add type restrictions to DSL Syntax](https://github.com/openfga/rfcs/pull/7)  |  #7 | Approved   |
```

Comment:
Instead of `Approved`, can we call it `Open` or `Open for Comment` to indicate it is ready for review and individual reviewer's suggestions may affect the status.